### ISSUE TITLE
[label/dropdown] right label and dropdown arrow overlaps

### ISSUE THREAD

**Issue by azmeuk** [fomantic/Fomantic-UI#2171]
# Bug Report
When using a dropdown for displaying a `<select>` input (in JS with `$("select").dropdown()`), with a label, the menu arrow of dropdowns overlaps with label icons.

## Steps to reproduce
```html
<div class="ui corner labeled input">
    <div class="ui fluid dropdown multiple selection">...<i class="dropdown icon"></i></div>
    <div class="ui corner label">
        <i class="asterisk icon"></i>
    </div>
</div>
```

## Expected result
I would expect the dropdown arrow to be on the right of the label.

## Actual result
The dropdown arrow and the label overlaps.

## Testcase
https://jsfiddle.net/azmeuk/9we7rcft/3/

## Screenshot (if possible)
![Screenshot 2021-12-05 at 18-16-01 Canaille - Interface de gestion des autorisations](https://user-images.githubusercontent.com/60163/144756601-2b98b65d-3ca0-4e3f-b3ef-62d09d1c1694.png)

## Version
2.8.8

**Comment by lubber-de** [comment]
Fixed by #2183 
See your adjusted jsfiddle here https://jsfiddle.net/lubber/5cf2j8k4/2/

**Comment by azmeuk** [comment]
Thank you!
